packages/client/tests/functional/interactive-transactions/tests.ts

```diff
@@ -196,19 +219,21 @@ testMatrix.setupTestSuite(({ provider }) => {
       })
     })
 
-    1
     await expect(result).rejects.toMatchObject({
       message: expect.stringContaining('Transaction API error: Transaction already closed'),
+      code: 'P2028',
+      clientVersion: '0.0.0',
     })
+
     await expect(result).rejects.toThrowErrorMatchingInlineSnapshot(`
```

Comment:
```suggestion
    await expect(result).rejects.toMatchPrismaErrorInlineSnapshot(`
```

My bad, missed this in my sanitation PR, but it will help with the diff.